style.css

```diff
@@ -27,10 +29,8 @@ body {
 
 .section-home {
   padding: 0 2.8rem;
-  text-align: center;
   background-image: url("/header-illsutration-mobile@2x.svg");
```

Comment:
Hello, @bienvenuushindi. After your review I opened my project on my local machine and Background images are visible.
<img width="828" alt="Screen Shot 2022-10-29 at 5 29 03 PM" src="https://user-images.githubusercontent.com/110017001/198831608-8d585002-c9a2-498d-b1c9-66b3860bb7e4.png">

Besides, the previous reviewer could also open this page with background images.

Reply:
Hi @haliljon, 
You are right in saying that. But this won't work everywhere as it is for me. To prevent such a bug you can add the dot `.` at the beginning of your background image URL. With this, your image will always display no matter the environment

![notice this](https://user-images.githubusercontent.com/26736582/198834077-1d95b976-fe1c-4b29-a315-557c620f8052.gif)

Reply:
Oh I see, thank you for correcting me. I'll keep that in mind. 🙏